x-pack/plugins/uptime/public/lib/alert_types/lazy_wrapper/validate_monitor_status.ts

```diff
@@ -10,7 +10,7 @@ import {
   AtomicStatusCheckParamsType,
   MonitorAvailabilityType,
   StatusCheckParamsType,
-} from '../../../../common/runtime_types/alerts';
+} from '../../../../common/runtime_types';
```

Comment:
you will also need to revert this, actually when you import it from index.ts file, it ends up bundling everything from that index file into the imported bundle.

Reply:
Thanks for pointing that out. I had consider reverting it and didn't think it would impact. I was wrong! Fixed this in https://github.com/elastic/kibana/pull/73528/commits/b1b8b780a11b438a571c08c933a97f7173523585.